lib/contracts/notes/game/fold.masm

```diff
@@ -38,6 +40,21 @@ proc.play_fold
 
     # adding 4 will give the slot of next player 
     push.4 add
+    # => [next_turn_index]
+
+    # find last player's index
+    push.NO_OF_PLAYERS_INDEX exec.account::get_item
+    drop drop drop
+    # => [no_of_players, next_turn_index]
+
+    # Calculate last_player_index = 52 + 13 * no_of_players + 0
+    push.13 mul
```

Comment:
Since 52 and 13 would always remain fixed so can you create constants for storing these values. Or use them if you have already created one.